docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -246,22 +300,72 @@ The `locale` prop of the `LocalizationProvider` component have been renamed `ada
 
 ## Component slots (custom sub-components)
```

Comment:
We should probably re-order the sections inside `Component slots`) 
Either alphabetically or thematically